categories.json

```diff
@@ -0,0 +1,639 @@
+[
+  {
+    "model": "ports.Category",
+    "pk": "amusements",
+    "fields": {
+      "name": "amusements"
```

Comment:
I wanted to provide some initial data which someone who tries to run the app on their system can use to populate the database.

But a python script can also generate this file, and so it can be removed.